src/plugins/administrative_templates/ui/platformmodel.h

```diff
@@ -23,8 +23,15 @@
 
 #include <QStandardItemModel>
 
+#include "../admx/supportedproduct.h"
```

Comment:
We need to add <memory.h> include to be able to compile this with gcc version 8.4.1 20200305.